product_state/README.rst

```diff
@@ -0,0 +1,110 @@
+=============
+Product State
+=============
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproduct--attribute-lightgray.png?logo=github
+    :target: https://github.com/OCA/product-attribute/tree/15.0/product_state
+    :alt: OCA/product-attribute
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/product-attribute-15-0/product-attribute-15-0-product_state
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/135/15.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module introduces the state field on product template and allows simple product life cycle:
+
+- draft: In Development
+- sellable: Normal
+- end: End of Lifecycle
+- obsolete: Obsolete
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To create a new state:
+
+#. Go to *Sales > Configuration > Products > Product States*.
+#. You can set its name and a description.
+
+To add a product to a state:
+
+#. Go to the product itself and edit.
+#. You can select the desired status in the list of buttons above the form.
+
+Known issues / Roadmap
+======================
+
+File data/product_state_data.xml will be moved to demo/product_state_demo.xml since version 15.0
```

Comment:
Please remove this line. The file was indeed moved in the 15.0 migration but it was reverted in https://github.com/OCA/product-attribute/commit/6079fdb26a6b792ff1c26966348c3f08441debf4.